bears/vcs/git/GitCommitBear.py

```diff
@@ -269,7 +336,9 @@ def check_issue_reference(self, body,
                               body_close_issue: bool=False,
                               body_close_issue_full_url: bool=False,
                               body_close_issue_on_last_line: bool=False,
-                              body_enforce_issue_reference: bool=False):
+                              body_enforce_issue_reference: bool=False,
+                              body_enforce_issue_open: bool=False,
+                              remote: str=None):
```

Comment:
We could possibly be able to do away with this particular setting, and use a brute-force method of checking over all remotes for an open issue with the issue number,
Coming up with an optimal algorithm to find the central repository could be a little troublesome though
(maybe you can think something on the lines of --set-upstream option, but I couldn't develop the idea, so I would recommend bruteforce)

Reply:
This setting must be removed.
I've said that many times.
Different people have different remote names for the same thing, and so `remote` can not be a setting which applies to everyone.
We can use `git remote` to get information we need, but not remote names.  if there are multiple remote URLs pointing at different repos, the bear needs to sanely figure out the right one.  Or *fail* because the users remotes are too strange.  Failing is better than allowing a remote name to be in the settings and forcing everyone to use the same remote names (or fail because they choose to have a different setup)